bin/logagent-setup.sh

```diff
@@ -20,6 +20,37 @@ while getopts ":i:u:g:" opt; do
     i)
       export TOKEN=$OPTARG
       ;;
+    j)
+      export JOURNALD=yes
+      ls /lib/systemd/system/systemd-journal-upload.service > /dev/null
+      if [ ! $? -eq 0 ]; then
+        echo
+        echo "journal-upload service not found. Attempting to install it..."
+        echo
+        which apt-get > /dev/null
```

Comment:
@radu-gheorghe a small hint: use [command](https://stackoverflow.com/questions/592620/how-can-i-check-if-a-program-exists-from-a-bash-script) instead of `which`.

Reply:
I didn't know about this one, thanks! Funny thing, `command` was used as a variable at some point 😛